alana/prompt.py

```diff
@@ -1,11 +1,27 @@
-from typing import List, Dict, Optional, Union, Literal, Any
-from alana.color import red, yellow
 import re
 import os
-from anthropic import Anthropic
-from anthropic.types import Message, MessageParam
+import httpx
+from typing import List, Dict, Optional, Union, Literal, Any, TypedDict, Unpack
+from anthropic import Anthropic, NotGiven
+from anthropic._types import Headers, Query, Body
+from anthropic.types import Message, MessageParam, message_create_params
```

Comment:
I think we should be importing MessageCreateParams not message_create_params

Reply:
We can also import `Metadata` directly to be more specific.

Reply:
Sounds good.